src/checks.py

```diff
@@ -159,8 +161,72 @@ def check_base_dir_in_zip(state: State) -> Optional[str]:
     return _check_sh(f"test -d {state.source_dir}")
 
 
+def _check_dircmp_expected_top_level_diffs(diff: filecmp.dircmp) -> [str]:
+    expected_top_level_git_only = {
```

Comment:
I think maybe I'd make this variable something more like: banned_from_release_package or something that says that it's not allowed in the release instead of about git

Reply:
Good point, doing.

Reply:
Not to mention, we don't even really care if these files are in the git tree, so this check should be separate from the diff stuff. I'll make that change.